src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="image imac"
+        class="card__image"
+      >
+      <span class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+      <span class="card__product-code">Product code: 195434</span>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <span class="card__rating-reviews">Reviews: 5</span>
+      </div>
+
+      <div class="card__prise">
+        <span class="card__prise-title">Price:</span>
+        <span class="card__prise-amount">$2,199</span>
+      </div>
+
+      <a
+        href="#"
+        class="card__link"
+        data-qa="hover">
+        Buy
+      </a>
```

Comment:
```suggestion
      <a
        href="#"
        class="card__link"
        data-qa="hover"
      >
        Buy
      </a>
```

let's format code according to[ style guide](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)